steps/app-ray-mobile-security/1.3.6/step.yml

```diff
@@ -0,0 +1,88 @@
+title: App-Ray Mobile Security
+summary: |
+  Mobile application security testing with App-Ray's engine via API
+description: |
+  Fully automated mobile application security testing identifies vulnerabilities, mitigate risks, comply with regulations whether you are developer, CISO, CEO.
+website: https://app-ray.co/
+source_code_url: https://github.com/App-Ray/bitrise-step-app-ray-mobile-security
+support_url: https://github.com/App-Ray/bitrise-step-app-ray-mobile-security/issues
+published_at: 2021-01-05T18:59:26.736352777+01:00
+source:
+  git: https://github.com/App-Ray/bitrise-step-app-ray-mobile-security.git
+  commit: 9ac4ec92debbacaec18cb86e7c360b80a6e39936
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+project_type_tags:
+- android
+- ios
+type_tags:
+- test
+toolkit:
+  bash:
+    entry_file: step.sh
+deps:
+  brew:
+  - name: curl
+  apt_get:
+  - name: curl
+is_requires_admin_user: true
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- base_url: null
+  opts:
+    description: Example https://demo.app-ray.co/
+    is_required: true
+    summary: "Your App-Ray base URL \nExample https://demo.app-ray.co/"
+    title: AppRay instance base URL
+- app_dir: null
+  opts:
+    description: Directory path to the app file you want to upload, usually like $BITRISE\_APK\_PATH
+      or $BITRISE\_IPA\_PATH Generated by XCode Archive & Export step or Android Build
+    is_expand: true
+    is_required: true
+    summary: Directory path to the app file you want to upload, usually like $BITRISE\_APK\_PATH
+      or $BITRISE\_IPA\_PATH
+    title: Dyrectory pattern to APK or IPA
+- appray_user: null
+  opts:
+    description: User used to run analysis
+    is_required: true
+    is_sensitive: true
+    summary: Your App-Ray username
+    title: App-Ray username
+- appray_passwd: null
+  opts:
+    description: Password
+    is_required: true
+    is_sensitive: true
+    summary: Your App-Ray password
```

Comment:
A little bit out of the scope of review, but the account password for the security analysis tool specified outside the service looks preposterous.